ext/RMagick/rmdraw.c

```diff
@@ -1067,239 +1067,28 @@ Draw_composite(int argc, VALUE *argv, VALUE self)
 
     // Retrieve the image to composite
     image = rm_cur_image(argv[4]);
-    (void) rm_check_destroyed(image);
+    comp_img = rm_check_destroyed(image);
 
     x = NUM2DBL(argv[0]);
     y = NUM2DBL(argv[1]);
     width  = NUM2DBL(argv[2]);
     height = NUM2DBL(argv[3]);
 
-    // The default composition operator is "Over".
+    cop = OverCompositeOp;
```

Comment:
I think this line is unnecessary.
```suggestion
```

Because, the variable was initialized with declaration.
```
CompositeOperator cop = OverCompositeOp;
```

Reply:
Removed the duplicate declaration